.github/ISSUE_TEMPLATE/bugs.yml

```diff
@@ -1,7 +1,8 @@
 name: Report bug
 description: Report a bug found
 title: "bug: "
-labels: ["bug", "ui"]
+labels: ["ui"]
```

Comment:
Should de UI label be automatically added? Not all bugs are UI-related, I think.